practice/00-HTML-CSS-basics/First exercise.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+<head>Hello World
```

Comment:
The content of the page must be inside the body tag. Read the HTML documentation for more info: https://docs.webplatform.org/wiki/html/tutorials